### PR TITLE
Explicitly define sending keys in VRF job for easier tracking

### PR DESCRIPTION
**Pull request by vreff**

### CODE REVIEW

**Review by makramkd — COMMENTED** [review]
A bit out of the loop on this, what do we mean by "easier tracking" here?

**Comment by vreff** [comment]
> A bit out of the loop on this, what do we mean by "easier tracking" here?

Similar to VRFv2/BHS, we want our "fromAddresses" to be defined in the job spec, rather than just picking them up automatically from the node, so that we can make sure all right keys are topped up.

File: core/cmd/ocr2vrf_configure_commands.go [hunk 1]
```diff
@@ -424,6 +427,10 @@ func createDKGJob(lggr logger.Logger, app chainlink.Application, args dkgTemplat
 }
 
 func createOCR2VRFJob(lggr logger.Logger, app chainlink.Application, args ocr2vrfTemplateArgs) error {
+	var sendingKeysString = fmt.Sprintf(`"%s"`, args.sendingKeys[0])
+	for x := 1; x < len(args.sendingKeys); x++ {
+		sendingKeysString = fmt.Sprintf(`%s,"%s"`, sendingKeysString, args.sendingKeys[x])
+	}
```
Comment (makramkd) [inline core/cmd/ocr2vrf_configure_commands.go:433]: I believe this is just a `strings.Join`:

```suggestion
	strings.Join(args.sendingKeys, ",")
```

You can probably pass this `Join` call inline below (line 442)

File: core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go [hunk 2]
```diff
@@ -442,6 +448,10 @@ fromBlock           = %d
 
 	t.Log("Creating OCR2VRF jobs")
 	for i := 0; i < numNodes; i++ {
+		var sendingKeysString = fmt.Sprintf(`"%s"`, sendingKeys[i][0])
+		for x := 1; x < len(sendingKeys[i]); x++ {
+			sendingKeysString = fmt.Sprintf(`%s,"%s"`, sendingKeysString, sendingKeys[i][x])
+		}
```
Comment (makramkd) [inline core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go:454]: Ditto here re: `strings.Join`

File: core/cmd/ocr2vrf_configure_commands.go (hunk 1 above)
Comment (vreff) [reply core/cmd/ocr2vrf_configure_commands.go:433]: Yeah I think I overengineered this a bit, I got worried about the TOML parsing it exactly correctly.

**Comment by vreff** [comment]
seeing this after making the above change


```
ocr2vrf_integration_test.go:495: 
        	Error Trace:	/home/runner/work/chainlink/chainlink/core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go:495
        	            				/home/runner/work/chainlink/chainlink/core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go:335
        	Error:      	Received unexpected error:
        	            	(16, 28): strconv.ParseUint: parsing "aBC4bc4724b388760aEd04b518f[344](https://github.com/smartcontractkit/chainlink/actions/runs/4307382369/jobs/7512428379#step:13:345)C1A993A315": value out of range
        	            	toml error on load
        	            	github.com/smartcontractkit/chainlink/core/services/ocr2/validate.ValidatedOracleSpecToml
        	            		/home/runner/work/chainlink/chainlink/core/services/ocr2/validate/validate.go:26
        	            	github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/ocr2vrf/internal_test.runOCR2VRFTest
        	            		/home/runner/work/chainlink/chainlink/core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go:494
        	            	github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/ocr2vrf/internal_test.TestIntegration_OCR2VRF_ForwarderFlow
        	            		/home/runner/work/chainlink/chainlink/core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go:335
        	            	testing.tRunner
        	            		/opt/hostedtoolcache/go/1.20.1/x64/src/testing/testing.go:1576
        	            	runtime.goexit
        	            		/opt/hostedtoolcache/go/1.20.1/x64/src/runtime/asm_amd64.s:1598
        	Test:       	TestIntegration_OCR2VRF_ForwarderFlow
```

Will probably revert to my "overengineered" implementation, it seems a strings.join is not good enough actually.

**Review by makramkd — APPROVED** [review]

**Pull request merged**